projects/schematics/src/wrapper-module/index_spec.ts

```diff
@@ -249,7 +249,9 @@ describe('Spartacus Wrapper Module Schematics: ng g @spartacus/schematics:wrappe
   });
 
   describe('Checkout and DP', () => {
-    it('Should order the imports in the wrapper and Spartacus features modules', async () => {
```

Comment:
Previously this test contained 4 assertions:
```ts
expect(spartacusFeaturesModule.print()).toMatchSnapshot();
expect(checkoutFeatureModule.print()).toMatchSnapshot();
expect(checkoutWrapperModule.print()).toMatchSnapshot();
expect(dpFeaturesModule.print()).toMatchSnapshot();
```

Now the first snapshot test is removed `expect(spartacusFeaturesModule.print()).toMatchSnapshot();` and replaced with explicit checking the contents of the SpartacusFeaturesModule - i.e. asserting whether the `DigitalPaymentsFeatureModule` is imported after the `CheckoutFeatureModule`.

It was done to avoid checking the whole contents of the SpartacusFeaturesModule. It's because this content is now going to change from release to release - as we introduce (and eventually remove after 122 months) the feature toggles.

The rest of 3 snapshot assertions (for other files) are now splitted to separate `it()` blocks for better readability of snapshots' descriptions.